README.md

```diff
@@ -479,6 +479,10 @@ Select when to upload container logs.
 
 The default is `on-error`.
 
+### `cli-version` (optional)
+
+If set to `1`, plugin will use `docker-compose` executable, else `docker compose` will be used.
```

Comment:
I believe that his wording makes it sound like version 2 is the default (I actually had to read the code to confirm it was not the case). I may suggest to word it differently:

```suggestion
If set to `2`, plugin will use `docker compose` to execute commands; otherwise it will default to version `1` using `docker-compose` instead.
```

Note that this wording also means that you will have to make a minor change to the logic of the code (see my other comment)